web/gen/generate_columns.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import json
+import re
+from pathlib import Path
+
+
+def generate_columns():
+    utils_path = Path("web/src/js/flow/utils.ts")
+
+    with open(utils_path, "r") as file:
+        content = file.read()
+
+    match = re.search(r"export const sortFunctions = {([\s\S]*?)};", content)
+    if not match:
+        raise ValueError("Could not find sortFunctions in utils.ts")
+
+    sort_functions_text = match.group(1)
+
+    columns = re.findall(r"^\s*(\w+):", sort_functions_text, re.MULTILINE)
```

Comment:
Personally, I'm not a big fan of this approach. Here's what I would have done:

1. Hardcode the two string arrays here and then define them in `web_columns.py` (backend) and, for instance, in `backends/const.ts` (frontend).
2. This way, we ensure the two arrays are generated and available on both the frontend and backend.
3. We can then refactor a bit `sortFunctions` by improving the typing and utilizing the newly created variable `AVAILABLE_COLUMNS`.

what do you think? :)